modules/installation/proc-add-custom-app-config-file-ocp-operator.adoc

```diff
@@ -0,0 +1,95 @@
+[id="proc-add-custom-app-config-file-ocp_{context}"]
+= Adding a custom application configuration file to {ocp-short} using the Operator
+
+A custom application configuration file is a ConfigMap that you can use to change the configuration of your {product} instance to meet the needs and preferences of your organization. If you are deploying your {product-short} instance on {ocp-brand-name}, you can use the {product} Operator to add a custom application configuration file to your {ocp-short} instance by creating the ConfigMap and referencing it in the {product-short} Custom Resource.
```

Comment:
```suggestion
A custom application configuration file is a `ConfigMap` object that you can use to change the configuration of your {product} instance. If you are deploying your {product-short} instance on {ocp-brand-name}, you can use the {product} Operator to add a custom application configuration file to your {ocp-short} instance, by creating the `ConfigMap` object and referencing it in the {product-short} custom resource (CR).
```
Couple of things:
- Suggested editing for minimalism to remove fluff in the first sentence.
- Suggested formatting of config map. Per the OCP guidelines and the RH supplementary style guide, for consistency across the org, we should either use the API object formatting if we're referring to a specific object ("`ConfigMap` object"), or it should be formatted as "config map".
- Suggested formatting of custom resource. Per the OCP guidelines and RHSG we should use "custom resource (CR)" for the first instance, and then you can use CR throughout the rest of the module etc if you like.

Links for ref:
- https://redhat-documentation.github.io/supplementary-style-guide/#custom-resource
- https://redhat-documentation.github.io/supplementary-style-guide/#config-map
- https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting